workbench/backend/src/app/query/query.controller.ts

```diff
@@ -4,27 +4,44 @@ import { AddressDTO, TagDTO } from '../../common/dto-models';
 import { FsmAPIClientFactory } from '../../common/fsm-api-client.factory';
 import { DTOName } from 'fsm-sdk/release/core/dto-name.model';
 
+interface QueryResponse<T> {
+  data: T[];
+}
+
 @Controller('api/query')
 export class QueryController {
 
   constructor(private factory: FsmAPIClientFactory) { }
 
   @Post()
   async query<T>(@Context() ctx: Context, @Body() { query }: { query: string }): Promise<T> {
-    return await this.factory.fromContext(ctx).query(query, Object.keys(this.factory.ALL_DTO_VERSIONS) as DTOName[])
-      .then(x => x.data)
-      .catch(e => { console.error(query); throw e; return undefined as any }) as T;
+
+
+    const coreApiClient = this.factory.fromContext(ctx);
+    const all_dto_versions = this.factory.ALL_DTO_VERSIONS
+    delete all_dto_versions["CrowdExecutionRecord"]
```

Comment:
This is a workaround and will be necessary until https://github.com/SAP/fsm-sdk/pull/26 is merged.